core/src/main/java/com/linecorp/armeria/common/ResponseHeadersBuilder.java

```diff
@@ -42,6 +42,21 @@
      */
     ResponseHeadersBuilder status(HttpStatus status);
 
+    /**
+     * Sets the <a href="https://datatracker.ietf.org/doc/html/rfc6265#section-4.1">set-cookie</a> header.\
+     */
+    ResponseHeadersBuilder cookie(Cookie cookie);
+
+    /**
+     * Sets the {<a href="https://datatracker.ietf.org/doc/html/rfc6265#section-4.1">set-cookie</a> header.\
+     */
+    ResponseHeadersBuilder cookies(Iterable<? extends Cookie> cookies);
+
+    /**
+     * Sets the <a href="https://datatracker.ietf.org/doc/html/rfc6265#section-4.1">set-cookie</a> header.\
```

Comment:
- Remove trailing `\`?
- `s/{<a href=/<a href=/g`

Reply:
Thank you for much guidance. 🙏